chromaui/addon-visual-tests
# Show outdated message when new test is available, and disable review button immediately

1. You can't accept changes when there is a next build
2. As soon as there's a new snapshot for the current story, we change the header to send you there.

## How to QA

1. Create a build with changes
2. Browse to the story with changes
3. Run second build:
  a. Review button should be disabled as soon as second build reaches the "snapshotting" state
  b. Header should prompt you to switch stories as soon as the story you are looking at is captured.

<details>
  <summary>📦 Published PR as canary version: <code>0.0.66--canary.87.fc95b12.0</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @chromaui/addon-visual-tests@0.0.66--canary.87.fc95b12.0
  # or 
  yarn add @chromaui/addon-visual-tests@0.0.66--canary.87.fc95b12.0
  ```
</details>
